### DISCUSSION TITLE
No response from GitHub support on ticket; can't verify educator status

### DISCUSSION THREAD

**Discussion by danielathome19** [community/community#67056] [GitHub Education] [answered]
### Hi everyone,

It's been nearly a month now that I've been waiting for a response from GitHub support on a ticket I put in for trying to verify my GitHub Education account. I teach high school CS and I'm a Ph.D. candidate; I've tried my school ID, work ID, school district work ID, paystubs, transcripts, course listings, and numerous permutations of each -- both applying as either a student or a teacher (which I was previously verified for) to no avail for either. I've been trying to reverify my account since the end of July, and I put in a ticket a month ago now with no response. I even tried contacting GitHub Enterprise to try a different support line and haven't gotten a confirmation email.

**Comment by muhumuzaa** [comment]
It’s like GitHub doesn’t really want to give us these packages. They make it so hard with so many twists with the of discouraging some applicant so they can just pay.

**Comment by danielathome19** [comment, marked as answer]
The one thing that finally ended up working was me opening a ticket through the mobile app, for some reason. Even after opening tickets about GitHub Enterprise, using their actual support system, and numerous email attempts.

So, if you run into this same issue, go to the GitHub Mobile app, and under settings, click "Get Help" and fill out a ticket there as well.
